docs/gettinginvolved/infrastructure/release-process.rst

```diff
@@ -52,6 +52,8 @@ Access
 - Access to Amazon `S3`_ using `gsutil`_ on your :doc:`Flocker development machine <vagrant>`.
   Set ``aws_access_key_id`` and ``aws_secret_access_key`` in the ``[Credentials]`` section of ``~/.boto``.
 
+- Access to `S3` and CloudFront via web.
```

Comment:
I am awaiting credentials so I can see how this works. If these can be links to something only accessible with the correct login details, then that would be helpful.